projects/fal/src/fal/api.py

```diff
@@ -75,6 +75,7 @@
 
 THREAD_POOL = ThreadPoolExecutor()
 
+
```

Comment:
Side note: linter wasn't tipped off about this, I'll need to add a rule...

Reply:
yes, my ruff did this all automatically on save :disappointed: